projects/igniteui-angular/src/lib/grids/grid/grid.component.ts

```diff
@@ -672,6 +672,7 @@ export class IgxGridComponent extends IgxGridBaseDirective implements GridType,
      */
     public clearGrouping(name?: string | Array<string>): void {
         this._gridAPI.clear_groupby(name);
+        this.calculateGridSizes();
         this.notifyChanges(true);
```

Comment:
notifyChanges can be removed since its main purpose is to force reflow calculation on the next detection cycle. But with this change the calculation is already done explicitly so there's no point to calculate again on the next detection cycle.